packages/devtools_app/lib/src/screens/debugger/program_explorer.dart

```diff
@@ -169,28 +167,28 @@ class _ProgramExplorerRow extends StatelessWidget {
           buffer.write('{');
         }
       }
-      if (param.required != null && param.required) {
+      if (param.required != null && param.required!) {
         buffer.write('required ');
       }
-      if (param.parameterType.name == null) {
-        buffer.write(_buildFunctionTypeText('Function', param.parameterType));
+      if (param.parameterType!.name == null) {
```

Comment:
pull param.parameterType! into local. Same for other cases where we find ourselves null asserting the same things multiple times